Python/Tests/Utilities.Python/PythonTestExplorerGridView.cs

```diff
@@ -78,7 +88,7 @@ public class PythonTestExplorerGridView : ListView {
                 var node = nodes[i];
                 var name = node.GetCurrentPropertyValue(AutomationElement.NameProperty) as string;
 
-                if (name.Equals(splitPath[depth], StringComparison.CurrentCulture)) {
+                if (name.Contains(splitPath[depth])) {
```

Comment:
NameProperty is now getting appended with strings like "Not Run". we can no longer use Equals

Reply:
You should mention that in a comment